index.js

```diff
@@ -373,11 +367,19 @@ class WriteStreamSync extends WriteStream {
   }
 }
 
-const writev = (fd, iovec, pos, cb) => {
-  const done = (er, bw) => cb(er, bw, iovec)
-  const req = new FSReqWrap()
-  req.oncomplete = done
-  binding.writeBuffers(fd, iovec, pos, req)
+let writev = fs.writev
+/* istanbul ignore next */
+if (!writev) {
+  const binding = process.binding('fs')
+  const writeBuffers = binding.writeBuffers
+  const FSReqWrap = binding.FSReqWrap || binding.FSReqCallback
+
+  writev = (fd, iovec, pos, cb) => {
+    const done = (er, bw) => cb(er, bw, iovec)
+    const req = new FSReqWrap()
+    req.oncomplete = done
+    binding.writeBuffers(fd, iovec, pos, req)
```

Comment:
@mikemimik It's used here

Reply:
That's `binding.writeBuffers`, above is pulling that function off so use as `writeBuffers` but then we're not using it. We're just accessing it from `binding`.

Reply:
Oic, that's right.  Yeah, we can delete the variable.